src/app/config.rs

```diff
@@ -70,7 +70,9 @@ impl Default for Config {
             results_kafka_topic: "uptime-results".to_owned(),
             configs_kafka_cluster: vec![],
             configs_kafka_topic: "uptime-configs".to_owned(),
-            redis_cluster_nodes: vec![],
+            redis_cluster_nodes: (0..6)
+                .map(|i| format!("redis://127.0.0.1:700{}", i))
+                .collect::<Vec<_>>(),
```

Comment:
Is this the normal default?

Reply:
Right, we should add a README at some point. You can run a redis cluster in sentry by setting `SENTRY_DEV_USE_REDIS_CLUSTER = True`. This ip address points to that cluster